hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -683,14 +703,14 @@ class DataObserver:
     ConjectureData object, primarily used for tracking
     the behaviour in the tree cache."""
 
-    def conclude_test(self, status, interesting_origin):
+    def conclude_test(self, status: Status, interesting_origin: None) -> None:
```

Comment:
Hmm, interesting that `interesting_origin` was picked up as None - implies that our tests are still weak.  (it's actually an `Optional[Tuple[...]]` which represents distinguishable errors)